coding-intro/coding-intro-core/core-concepts/think-like-a-coder.md

```diff
@@ -0,0 +1,49 @@
+---
+author: abdullah
+
+tags:
+  - discussion
+
+type: normal
+
+category: discussion
```

Comment:
here too, making this insight a `discussion` will display the "Give it a try" button, but there is nothing to try here

Reply:
Trying to make this a discussion insight as well.